NOTICE.txt

```diff
@@ -392,6 +347,10 @@ THE SOFTWARE.
 
 ---
 
+## dynamic-virtualized-list
```

Comment:
Unsure if it's https://github.com/mattermost/dynamic-virtualized-list or https://github.com/dwqs/react-virtual-list

Reply:
It's this: https://github.com/mattermost/dynamic-virtualized-list

You can find the package names in `package.json`. If they are URLs then that tells you which one it is (as in this case). If they are just names, then you can search on https://npmjs.com to find the package by it's name, and then the link to Github will be there.

Reply:
@grundleborg Thanks, added